pkg/registry/apps/apps.go

```diff
@@ -40,11 +44,19 @@ func ProvideRegistryServiceSink(
 		RestConfigGetter: cfgWrapper,
 		APIRegistrar:     registrar,
 	}
-	runner, err := runner.NewAPIGroupRunner(cfg, playlistAppProvider)
+
+	var apiGroupRunner *runner.APIGroupRunner
+	var err error
+	if features.IsEnabledGlobally(featuremgmt.FlagInvestigationsBackend) {
```

Comment:
@toddtreece FYI, I think we should look into having a generic FF here for enabling various APIs, similar to what `kube-apiserver` has in `--runtime-config`.

Reply:
@radiohead i think the main issue with a generic toggle is that feature toggles are booleans, so i'm not sure how we could do that unless we enable a group of them with a single toggle

we could use apienablement's `--runtime-config` for this purpose, and we probably should move to that in code and have the existing feature toggles add the API to `runtime-config`. the primary issue with using `runtime-config` directly in HG is that feature toggles have additional plumbing that allow us to control the roll out of new features, so we'd need to add something similar for APIs enabled via `--runtime-config` if we expose that directly via the grafana config